webui/src/docs/rest-api.jsx

```diff
@@ -1,244 +1,540 @@
+            <li>Create a draft record: use the
+                <a href={`#create-draft`}> Create draft record </a> function</li>
+            <li>Upload the files into the draft record. You will have to use
+                one HTTP request per file. Use the
+                <a href={`#upload-file`}> Upload file </a> function</li>
+            <li>Set the complete metadata and publish the record. Use the
+                <a href={`#publish-draft`}> Publish draft </a> function</li>
+        </ol>
+
+
+        <h2>The HTTP API</h2>
+
+        <h3 id="list-all-communities">List all communities</h3>
+        <p>List all the communities, without any filtering.</p>
+        <ul>
+            <li><p>HTTP method: GET</p></li>
+            <li><p>URL path: /api/communities</p></li>
+            <li><p>Required parameters: access_token</p></li>
+            <li><p>Returns: the list of communities (in JSON format) or an error message.</p></li>
+        </ul>
+        <Example>
+            curl https://$HOSTNAME/api/communities/?access_token=$ACCESS_TOKEN
+            <Returns>
+            {{
+              "hits": {
+                "hits": [
+                  {
+                    "created": "Tue, 18 Oct 2016 08:30:47 GMT",
+                    "description": "The big Eudat community. Use this community if no other is suited for you",
+                    "id": "e9b9792e-79fb-4b07-b6b4-b9c2bd06d095",
+                    "links": {
+                      "self": "https://vm0045.kaj.pouta.csc.fi/api/communities/e9b9792e-79fb-4b07-b6b4-b9c2bd06d095"
+                    },
+                    "logo": "/img/communities/eudat.png",
+                    "name": "EUDAT",
+                    "updated": "Tue, 18 Oct 2016 08:30:47 GMT"
+                  },
+                  "...",
+                ],
+                "total": 11
+              },
+              "links": {
+                "self": "https://vm0045.kaj.pouta.csc.fi/api/communities/"
+              }
+            }}
+            </Returns>
+        </Example>
+
+        <h3 id="get-community-schema">Get community schema</h3>
+        <p>Retrieves the JSON schema of records approved by a specific community.</p>
+        <ul>
+            <li><p>HTTP method: GET</p></li>
+            <li><p>URL path: /api/communities/$COMMUNITY_ID/schemas/last</p></li>
+            <li><p>Required parameters: access_token</p></li>
+            <li><p>Returns: the JSON schema, embedded in a JSON object, or an error message.</p></li>
+        </ul>
+        <Example>
+            curl https://$HOSTNAME/api/communities/$COMMUNITY_ID/schemas/last?access_token=$ACCESS_TOKEN
+            <Returns>
+            {{
+              "community": "e9b9792e-79fb-4b07-b6b4-b9c2bd06d095",
+              "draft_json_schema": {
+                "$ref": "https://vm0045.kaj.pouta.csc.fi/api/communities/e9b9792e-79fb-4b07-b6b4-b9c2bd06d095/schemas/0#/json_schema",
```

Comment:
What about creating a fake simple hostname which we keep consistent through all the examples.
```sh
 <Example>export HOSTNAME='trng-b2share.eudat.eu'</Example>
...
"$ref": "https://trng-b2share.eudat.eu/api/communities/e9b9792e-79fb-4b07-b6b4-b9c2bd06d095/schemas/0#/json_schema",
```
It would IMHO be more clear.